redisinsight/ui/src/utils/formatters/bufferFormatters.ts

```diff
@@ -141,6 +141,11 @@ const hexToBuffer = (data: string): RedisResponseBuffer => {
 const bufferToJava = (reply: RedisResponseBuffer) => {
   const stream = new ObjectInputStream(bufferToUint8Array(reply))
   const decoded = stream.readObject()
+
+  if (typeof(decoded) !== 'object') {
```

Comment:
Add space between typeof and (decoded) and remove semicolon in the on of the next line, please